composeml/label_maker.py

```diff
@@ -122,16 +76,39 @@ def search(self, df, minimum_data, num_examples_per_instance, gap, verbose=True,
         df = self._preprocess(df)
 
         assert_valid_offset(minimum_data)
-        assert_valid_offset(self.window_size)
         assert_valid_offset(gap)
 
-        df_to_labels = on_slice(
-            self.labeling_function,
-            min_data=minimum_data,
-            window=self.window_size,
-            n_examples=num_examples_per_instance,
-            gap=gap,
-        )
+        def df_to_labels(df):
+            name = self.labeling_function.__name__
+            labels = pd.Series(name=name)
+
+            df = df.loc[df.index.notnull()]
+            df.sort_index(inplace=True)
+
+            if df.empty:
+                return labels.to_frame()
+
+            cutoff_time = offset_time(df.index, minimum_data)
+
+            for example in range(num_examples_per_instance):
+                df = df[cutoff_time:]
+
+                if df.empty:
+                    break
+
+                time = offset_time(df.index, self.window_size)
```

Comment:
is it possible to give this variable a more descriptive name? perhaps `window_end`

Reply:
Renamed variable in [this](https://github.com/FeatureLabs/compose-ml/pull/26/commits/3faae1725789efb5eca283737381b3151a2ffb5b#diff-22bf41f653d21f0ca3dd15dadf5e01e9R99) commit.